bin/prettier.js

```diff
@@ -13,13 +13,15 @@ const argv = minimist(process.argv.slice(2), {
     "single-quote",
     "trailing-comma",
     "bracket-spacing",
+    // See https://github.com/chalk/supports-color/#info
+    "color",
     "version",
     "debug-print-doc",
     // Deprecated in 0.0.10
     "flow-parser"
   ],
   string: [ "print-width", "tab-width", "parser" ],
-  default: { "bracket-spacing": true, parser: "babylon" },
+  default: { color: true, "bracket-spacing": true, parser: "babylon" },
```

Comment:
The [supports-color](https://github.com/chalk/supports-color#info) package (a sub sub dependency) looks directly at `process.argv`. The reason it is listed here is to avoid "Ignored unkown option: --no-color" warnings.